packages/api/src/types/ipfs.ts

```diff
@@ -21,6 +21,30 @@ export type IpfsSpaceContent = {
   links: string[] | NamedLink[]
 }
 
+//Ref: https://github.com/PLhery/node-twitter-api-v2/blob/master/src/types/v2/tweet.definition.v2.ts
```

Comment:
We have copypasted types. @subsocial/utils includes to api, so you can import and export types here from utils